pages/about.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/about_styles.css">
 </head>
+
 <body>
-  
+    <section class="container">
+        <div id="purple_1_header">
```

Comment:
I'm not seeing what the `purple_...` id-labelled divs are intended for. Can these be removed to clean up the markup?